lib/Doctrine/ORM/Mapping/Driver/XmlDriver.php

```diff
@@ -39,6 +41,13 @@ class XmlDriver extends FileDriver
      */
     public function __construct($locator, $fileExtension = self::DEFAULT_FILE_EXTENSION)
     {
+        if (! function_exists('simplexml_load_string')) {
```

Comment:
Why not use https://www.php.net/manual/en/function.extension-loaded.php here?

Reply:
No strong opinions here. 🤷🏻‍♂️